src/modals/collectible-modal/components/collectible-properties/utils/get-royalties.util.ts

```diff
@@ -0,0 +1,16 @@
+import { isNonEmptyArray } from '@apollo/client/utilities';
+import { BigNumber } from 'bignumber.js';
+
+export const getRoyalties = (royalties: { amount: number; decimals: number }[]) => {
+  if (!isNonEmptyArray(royalties)) {
+    return null;
+  }
+
+  const { amount, decimals } = royalties[0];
+
+  const amountBN = new BigNumber(amount ?? 0);
+
+  const result = `${Number(amountBN.dividedBy(10 ** decimals).multipliedBy(100))}%`;
```

Comment:
are you sure about this formula? looks like we can avoid `multipliedBy`

Reply:
yes, I'm sure :)

Reply:
yes, it is a common solution for getting the correct percent value from that data, we have used it before